backend/src/Designer/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -904,9 +904,15 @@ public Stream GetProcessDefinitionFile()
 
         public Definitions GetDefinitions()
         {
-            Stream processDefinitionStream = GetProcessDefinitionFile();
+            using Stream processDefinitionStream = GetProcessDefinitionFile();
+            using MemoryStream memoryStream = new MemoryStream();
```

Comment:
Do we need memory stream here?

Reply:
Looks like we do not! Thanks for noticing 😄 Removed it now.